### PR TITLE
/download/desktop 24.04 copy updates

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied copy updates as per [doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13722.demos.haus/download/desktop
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the needed changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10051

### CODE REVIEW

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -2,233 +2,239 @@
...
+                    <ul class="p-list--divided">
+                        <li class="p-list__item is-ticked">New Desktop installer with support for autoinstall</li>
+                        <li class="p-list__item is-ticked">New App Center and Firmware Updater applications</li>
+                        <li class="p-list__item is-ticked">GNOME 46 with support for quarter screen tiling</li>
+                        <li class="p-list__item is-ticked">Advanced Active Directory Group Policy Object support for Ubuntu Pro users</li>
+                        <li class="p-list__item is-ticked">Experimental support for TPM-backed Full Disc Encryption and ZFS encryption</li>
+                    </ul>
+                    <hr class="p-rule" />
+                    <p>
+                        <a href="https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668">{{ releases.lts.full_version }} release notes&nbsp;&rsaquo;</a>
```
Comment (petesfrench) [inline templates/download/desktop/index.html:126]:
```suggestion
                        <a href="https://discourse.ubuntu.com/t/noble-numbat-release-notes/39890">{{ releases.lts.full_version }} release notes&nbsp;&rsaquo;</a>
```

**Review by petesfrench — APPROVED** [review]
LGTM

**Comment by juanruitina** [comment]
Link to release notes is wrong, but I noticed Pete already picked it up :)

Please, also remove the abbr in the Download CTA, I finally agreed with Lyubo to get rid of it. Acronym is spelled out above anyways. Thank you 🙏

**Comment by juanruitina** [comment]
LGTM!

**Pull request merged**